pynws/nws.py

```diff
@@ -64,6 +65,15 @@ async def get_points(self):
             self.fire_weather_zone = properties.get("fireWeatherZone").split("/")[-1]
         return properties
 
+    async def get_gridpoints(self):
```

Comment:
What do you think of this method name?

Reply:
It is descriptive to the NWS API, but it is confusing what the difference between `get_gridpoints` vs `get_gridpoints_forecast`.

I'm inclined to keep your current method name unless you have a better thought.  Maybe we can add  more distinguishing docstring in `get_gridpoints_forecast` like

```
"Return daily forecast from grid."""
```

Reply:
I don't care for the current name. Here are some possible names. 
* get_gridpoints_all_data
* get_gridpoints_data
* get_gridpoints_raw_data

I think "gridpoints" in all names should probably be "gridpoint", but that would be a major change and probably not worth it.

Reply:
Looking at this again, nearly all the methods have a tight naming correspondence to the underlying API endpoint name, hence "gridpoints".  In this view, the current method name makes sense for now.

We could consider deprecating the old names and replace with more descriptive names in the future.  This would be an acceptable breaking change if it makes sense.

Reply:
I would suggest something like 'get_forecast_all`, `get_forecast_daily`, and `get_forecast_hourly`

Reply:
I like those names. We could keep the old names as aliases to the new names to avoid the breaking change for now--deprecate later.

Reply:
> I like those names. We could keep the old names as aliases to the new names to avoid the breaking change for now--deprecate later.

Let's keep this PR small by not doing it here.  We can start with this method here.

Reply:
The new method has been renamed to get_forecast_all